store/migration/dev/20220715000000__primary_key.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE idx ADD COLUMN "primary" BOOLEAN NOT NULL DEFAULT false;
```

Comment:
So the ADD CONSTRAINT will fail if we do not set DEFAULT false first?

Reply:
In reality the ADD COLUMN NOT NULL will fail because NOT NULL